File: BJ/손익분기점/1.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    
    
    
    // freopen("input.txt", "r", stdin);
    
   
    int a,b,c;


    int answer=-1;
    long double cal;
    cin >>  a >> b >> c ;
    if(c<=b)
    {
        
    }
    else
    {
        cal=(long double)a/(long double)(c-b);
        answer=ceil(cal);
        if(cal==answer)
        {
            answer++;
        }


    }
    cout << answer << endl;


}

File: BJ/DP/연속합/1.py
import sys

# f = open("input.txt", 'r')
# line = f.readline()

line = input()
# line = f.readline()

line = line.replace("\n","")
N=int(line)

answer = [0] * N

array = list(map(int, sys.stdin.readline().split())) 

answer[0] = array[0]
result=answer[0]

for i in range(1,N):
    answer[i]=max(answer[i-1]+array[i],array[i])
    result=max(answer[i],result)

print(result)